tests/webapp/api/test_performance_data_api.py

```diff
@@ -632,6 +632,7 @@ def test_perfcompare_results_multiple_runs(
         suite=suite,
```

Comment:
I would be nice if you could change the push time on both pushes (different times) so that we can clearly see how the graph link changes based on the time.